src/main/java/gregtechfoodoption/recipe/chain/BreadsChain.java

```diff
@@ -243,7 +237,10 @@ public static void init() {
         ModHandler.addShapelessRecipe("gtfo_sandwich_veggie", SANDWICH_VEGGIE.getStackForm(), PRESLICED_BREAD, CUCUMBER_SLICE, CUCUMBER_SLICE, TOMATO_SLICE, TOMATO_SLICE, ONION_SLICE, ONION_SLICE);
         ModHandler.addShapelessRecipe("gtfo_sandwich_cheese", SANDWICH_CHEESE.getStackForm(), PRESLICED_BREAD, CHEDDAR_SLICE, CHEDDAR_SLICE, CHEDDAR_SLICE, CHEDDAR_SLICE);
         ModHandler.addShapelessRecipe("gtfo_sandwich_bacon", SANDWICH_BACON.getStackForm(), PRESLICED_BREAD, BACON, BACON, BACON, BACON, BACON, BACON, BACON, BACON);
-        ModHandler.addShapelessRecipe("gtfo_sandwich_steak", SANDWICH_STEAK.getStackForm(), PRESLICED_BREAD, GTFOMaterialHandler.MeatIngot.getItemStack(), CHEDDAR_SLICE, CHEDDAR_SLICE);
+        ModHandler.addShapelessRecipe("gtfo_sandwich_steak_from_meat", SANDWICH_STEAK.getStackForm(), PRESLICED_BREAD, GTFOMaterialHandler.MeatIngot.getItemStack());
+        ModHandler.addShapelessRecipe("gtfo_sandwich_steak_from_steak", SANDWICH_STEAK.getStackForm(), PRESLICED_BREAD, new ItemStack(Items.COOKED_BEEF));
+        ModHandler.addShapelessRecipe("gtfo_large_sandwich_steak_from_steak", SANDWICH_LARGE_STEAK.getStackForm(), PRESLICED_BAGUETTE, new ItemStack(Items.COOKED_BEEF), new ItemStack(Items.COOKED_BEEF), new ItemStack(Items.COOKED_BEEF));
+        ModHandler.addShapelessRecipe("gtfo_large_sandwich_steak_from_meat", SANDWICH_LARGE_STEAK.getStackForm(), PRESLICED_BAGUETTE, GTFOMaterialHandler.MeatIngot.getItemStack(), GTFOMaterialHandler.MeatIngot.getItemStack(), GTFOMaterialHandler.MeatIngot.getItemStack());
```

Comment:
Large sandwiches are gated behind the cuisine assembler in order to make it more desirable, so I would prefer not to have these recipes.